apps/feedback/utils.py

```diff
@@ -1,17 +1,23 @@
 # -*- coding: utf-8 -*-
 
 from django.contrib.contenttypes.models import ContentType
+from guardian.shortcuts import get_objects_for_user
 
+from apps.authentication.models import OnlineUser as User
 from apps.events.models import Event
-from apps.events.utils import get_group_restricted_events
 from apps.feedback.models import FeedbackRelation
 
 
-def has_permission(feedback_relation, user):
+def has_permission(feedback_relation, user: User):
     content_object = feedback_relation.content_object
 
     if isinstance(content_object, Event):
-        return content_object in get_group_restricted_events(user)
+        event: Event = content_object
+        if not user.is_authenticated:
+            return False
+        if user.is_superuser:
+            return True
+        return user.has_perm('events.change_event', event)
 
```

Comment:
This will actually return true for a superuser @oleast, so the above line is unnecessary 😄